report_thesis/src/sections/proposed_approach/testing_validation.tex

```diff
@@ -47,76 +59,142 @@ \subsection{Validation and Testing Procedures for Model Evaluation}\label{subsec
+By general representativeness, we mean ensuring that the test set reflects the general distribution of the dataset without being skewed by extreme values.
+This balance is essential for accurately assessing the model's performance on typical data points.
+
+Our method for handling extreme values ensures that the test set does not include samples outside the range seen in the training set.
+This approach addresses several critical challenges and represents a deliberate trade-off to improve the reliability of our model evaluation:
+
+Firstly, it mitigates the risk of uneven distribution of extreme values, which can disproportionately affect model performance metrics.
+If extreme values are unevenly distributed between the training and test sets, the evaluation of the model can be heavily skewed, leading to unreliable and misleading performance metrics.
+By redistributing extreme values evenly across the training folds, we ensure a more balanced and fair assessment of the model's capabilities during cross-validation.
 
-By following this procedure, we ensure that our cross-validation is robust against data leakage, maintains the integrity of grouped targets, and carefully handles extreme values to improve the representativeness of the training set.
+Secondly, this trade-off ensures a more stable and reliable assessment of the model's performance on typical data points.
+Although the test set may be less representative of the full range of data, particularly concerning the rare extreme values, the evaluation focuses on the model's ability to generalize from the training data to new data within the same distribution range.
+This is prudent because, in many practical applications, the reliability of the model on typical data points is more critical than its performance on rare extremes.
+By using cross-validation in conjunction with a separate test set, we ensure that the model is robust and performs well under typical conditions.
+
+However, it is important to recognize the limitation of this approach: excluding extreme values from the test set means we can only confidently assess the model's performance within the test set's range.
+If predictions fall outside this range, we cannot reliably assert their accuracy, as the model has not been fully evaluated on such data.
+This could potentially reduce the model's usefulness in scenarios where predictions on extreme values are critical.
+Nevertheless, cross-validation partially mitigates this issue by allowing the model to learn from extreme values during the training and validation phases, thereby improving its robustness.
+Metrics obtained from cross-validation provide a broader view of the model's accuracy and robustness across the entire dataset, including extreme values.
```

Comment:
I am honestly not sure what to think about this bit here. While its extremely well written, I feel like we are repeating what has already been said several times, but in a different order.
What is the goal with this part? Is it to summarize? If yes, then I almost feel like the summary is as long as the text that first described these concepts.
If no, then what is being added here that the reader didnt already know from what was read not too long ago?

I might be wrong, but it almost feels like we are so afraid that its not obvious what we are trying to do that we overcompensate and lose a lot of precision in the process.

I feel like you could take some of these points made here and perhaps bolster the arguments made earlier and then cut down on a lot of text.

What do you think?